dynawo/sources/Models/Modelica/Dynawo/Electrical/Shunt.mo

```diff
@@ -0,0 +1,43 @@
+within Dynawo.Electrical;
+
+/*
+* Copyright (c) 2015-2019, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+*/
+
+model Shunt "Shunt element with voltage dependent reactive power, constant susceptance"
+
```

Comment:
We could create an Icon for the shunt, what do you think?

Reply:
For sure. 
What do you think of the proposal below:

Reply:
ok for me ;)

Reply:
ok! :)

Reply:
Shall I do anything about that?

Reply:
yes please, it is in the view given by this icon in OM (you can draw lines etc)
![image](https://user-images.githubusercontent.com/25203289/91728215-16075f80-eba3-11ea-8303-ee4042812b97.png)

Reply:
Ah, I thought it will be something in the AdditionalIcons that will be extended within the Shunt model, e.g. AdditionalIcons.Shunt

Reply:
yes in the AdditionalIcon you have to add a new one :)

Reply:
oki, I'll do.

Reply:
Thanks Maria, it look nice but really small. Could you enlarge it a bit, so that it takes all the lower part of the page (a bit like the load icon?) Thanks a lot

Reply:
done